piv/build.gradle

```diff
@@ -3,6 +3,8 @@ apply plugin: 'java-library'
 dependencies {
     api project(':core')
 
+    api 'org.slf4j:slf4j-api:2.0.7'
```

Comment:
I think this can be an `implementation` dependency? I didn't see it in any public members or method signatures, but I could have missed some of course.